src/libraries/System.Drawing.Common/tests/System.Drawing.Common.Tests.csproj

```diff
@@ -3,6 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
+    <IgnoreForCI Condition="'$(TargetsBrowser)' == 'true'">true</IgnoreForCI>
```

Comment:
We need to use `TargetOS` here because the test project is not cross compiled for `Browser` -- sorry about that.

Reply:
```suggestion
    <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
```